BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudRelay.cs

```diff
@@ -29,8 +29,8 @@ public struct RelayConnectOptions
 
     public class BrainCloudRelay
     {
-        public const int TO_ALL_PLAYERS = 131;
-        public const int MAX_PLAYERS = 128;
+        public const ulong TO_ALL_PLAYERS = 0x000000FFFFFFFFFF;
+        public const int MAX_PLAYERS = 40;
```

Comment:
Just confirming you still want 40 MAX_PLAYERS, and wasn't just testing something out

Reply:
Nope, max 40 players is correct.